examples/react-datastore/src/datastore/generated/schema.json

```diff
@@ -3,6 +3,7 @@
     "name": "Todo",
     "version": 1,
     "type": "object",
+    "primaryKey": "_id",
```

Comment:
This is not valid property. we setup this field in code from fields. Gianluca can confirm

Reply:
Maybe we should have it on generator and remove it in code but this will allow people to have invalid values